tests/BUILD

```diff
@@ -113,8 +123,8 @@ py_test(
     name = "hello_licenses_test",
     srcs = ["hello_licenses_test.py"],
     data = [
-        ":hello_licenses.json",
         ":hello_cc_copyrights.txt",
+        ":hello_licenses.json",
```

Comment:
I debated reverting these off-target changes that were made by buildifier, but... they were made by buildifier, so probably best to leave them in anyway.